.github/workflows/tests_linux.yml

```diff
@@ -644,7 +644,9 @@ jobs:
         run: |
           python -m pip install coverage
           python -m coverage combine .coverage-python*
-          python -m coverage xml -o coverage-${{ github.job }}.xml
+          # Added cov xml -i to ignore "No source for code" random errors
+          # https://stackoverflow.com/questions/2386975/no-source-for-code-message-in-coverage-py
+          python -m coverage xml -i -o coverage-${{ github.job }}.xml
 
```

Comment:
Hi guys, you're sure we won't miss important errors with:
`-i, --ignore-errors   Ignore errors while reading source files`

Reply:
@tomlqc I'd argue that if there are any errors in uploading all coverage reports (due to missing some important errors at this step), then the codecov CI report will fail because of the coverage reduction in the code base. We can always revert this change if we find out this patch is defective.

Reply:
@tomlqc I presume that in the event of any errors during the export of coverage reports, the codecov CI report may fail due to a reduction in code base coverage. I didn't see any issue with > 10-time triggering CIs that may be a good sign. If we discover that this patch is defective, we have the option to revert this change at anytime.